### PR TITLE
Remove compliance server samples

### PR DESCRIPTION
**Pull request by rayhayes**
Remove compliance server samples given that it isn't supported in WMF 5.

### CODE REVIEW

File: Examples/Sample_xDscWebServiceRegistration.ps1 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# DSC configuration for Pull Server and Compliance Server
```
Comment (NarineM) [inline Examples/Sample_xDscWebServiceRegistration.ps1]: There is an example to remove pull service that also needs to be updated:
Sample_xDscWebServiceRemoval.ps1

**Comment by KarolKaczmarek** [comment]
Please update also

**Comment by KarolKaczmarek** [comment]
Please update also:
Examples/Sample_xDscWebServiceRemoval.ps1 
/Tests/Integration/MSFT_xDSCWebService.xxx.ps1

File: Examples/Sample_xDscWebServiceRegistration.ps1 (hunk 1 above)
Comment (rayhayes) [inline Examples/Sample_xDscWebServiceRegistration.ps1]: Done.

**Pull request merged**